src/main/com/fulcrologic/rad/type_support/date_time.cljc

```diff
@@ -148,8 +148,9 @@
 (>defn html-date-string->local-date
   "Convert a standard HTML5 date input string to a local date"
   [s]
-  [string? => date?]
-  (ld/parse s))
+  [string? => (? date?)]
```

Comment:
breaking change....but I guess there really isn't a value for a "cleared" date input. Technically your change is correct, just need to consider (since this is published code) if we want a new function written correctly, or this patch. This patch is probably better. Tests should be fixed.

Similar longer comment on the other ones below...

Reply:
Another way is to fix the callers of these functions to skip the call if input is `""`. So instead of changing these functions in fulcro-rad I can patch callers of these in fulcro-rad-semantic-ui to not call them with `""`. That is more non-breaking, but seems less clean. If you'd prefer that I can do that.

Reply:
I feel the same way about it as you do. This one would throw before on "", so I think your patch is fine. No one can be relying on any good behavior of passing ""